src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -1,161 +1,54 @@
-import React, { useCallback, useEffect, useState } from "react";
+import React, { useEffect } from "react";
+import { MediaPlayer } from "components/attendee/MediaPlayer";
 import { useBackgroundGradient } from "components/attendee/useBackgroundGradient";
-import { useVideoHuddle } from "components/attendee/VideoHuddle/useVideoHuddle";
 
-import {
-  DEFAULT_REACTIONS_MUTED,
-  IFRAME_ALLOW,
-  JAZZBAR_TABLES,
-} from "settings";
-
-import { unsetTableSeat } from "api/venue";
+import { JAZZBAR_TABLES } from "settings";
 
 import { JazzbarVenue } from "types/venues";
-import { VenueTemplate } from "types/VenueTemplate";
 
-import { convertToEmbeddableUrl } from "utils/embeddableUrl";
 import { WithId } from "utils/id";
 
 import { useAnalytics } from "hooks/useAnalytics";
-import { useExperience } from "hooks/useExperience";
-import { useRelatedVenues } from "hooks/useRelatedVenues";
-import { useShowHide } from "hooks/useShowHide";
-import { useUpdateTableRecentSeatedUsers } from "hooks/useUpdateRecentSeatedUsers";
 import { useUser } from "hooks/useUser";
 
 import { RenderMarkdown } from "components/organisms/RenderMarkdown";
 
-import { TableComponent } from "components/molecules/TableComponent";
-import { TablesUserList } from "components/molecules/TablesUserList";
-
-import { BackButton } from "components/atoms/BackButton";
+import { Loading } from "components/molecules/Loading";
+import { TableGrid } from "components/molecules/TableGrid";
 
-import "./JazzBar.scss";
+import styles from "./JazzBar.module.scss";
 
 interface JazzProps {
   venue: WithId<JazzbarVenue>;
 }
 
 export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
-  const { isShown: showOnlyAvailableTables } = useShowHide();
-  const { parentVenue } = useRelatedVenues({ currentVenueId: venue.id });
-  const embedIframeUrl = convertToEmbeddableUrl({
-    url: venue.iframeUrl,
-    autoPlay: venue.autoPlay,
-  });
   const analytics = useAnalytics({ venue });
 
   useBackgroundGradient();
 
-  // @debt what does it do anyway? cache? no useSelector here so probably connected to useUpdateTableRecentSeatedUsers
-  useExperience(venue.name);
-
   const jazzbarTables = venue.config?.tables ?? JAZZBAR_TABLES;
 
-  const [seatedAtTable, setSeatedAtTable] = useState<string>();
-  const { userWithId, userId } = useUser();
-
-  useUpdateTableRecentSeatedUsers(
-    VenueTemplate.jazzbar,
-    seatedAtTable && venue?.id
-  );
-
-  const isReactionsMuted = venue.isReactionsMuted ?? DEFAULT_REACTIONS_MUTED;
-
-  const { hide: enableUserAudio, show: disableUserAudio } = useShowHide(
-    isReactionsMuted
-  );
-
-  useEffect(() => {
-    if (isReactionsMuted) {
-      disableUserAudio();
-    } else {
-      enableUserAudio();
-    }
-  }, [isReactionsMuted, disableUserAudio, enableUserAudio]);
+  const { userId } = useUser();
```

Comment:
There is a lighter hook for this, one that doesn't spend time reading profile info from DB.

```suggestion
  const { userId } = useUserId();
```